modules/ossm-install-cert-manager.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-install-cert-manager_{context}"]
+== Install cert-manager
+
+To install cert-manager, follow these steps:
+
+//. Install cert-manager:
+//+
+//[source, terminal]
+//----
+//helm install cert-manager jetstack/cert-manager \
+//    --namespace cert-manager \
+//    --create-namespace \
+//    --version v1.11.0 \
+//    --set installCRDs=true
+//----
+
+. Provision certificates:
++
+[source, terminal]
+----
+oc new-project istio-system
+oc apply -f deploy/examples/cert-manager/selfsigned-ca.yaml -n istio-system
+----
+
+. Install cert-manager `istio-csr` service:
++
+[source, terminal]
+----
+helm install -n cert-manager cert-manager-istio-csr \
+    jetstack/cert-manager-istio-csr -f deploy/examples/cert-manager/istio-csr-helm-values.yaml
```

Comment:
This line references another yaml file from our example on the istio-operator repo. @pbajjuri20, shouldn't we include the content of this yaml as well?

Reply:
Yes, It is also required the specific yaml